### PR TITLE
Update welding examples

### PR DESCRIPTION
**Pull request by vhirtham**
## Changes

_Describe changes in this PR_

## Related Issues

Closes # (add issue numbers)

## Checks

- [ ] updated CHANGELOG.md
- [ ] updated tests
- [ ] updated doc/
- [ ] update example/tutorial notebooks

### CODE REVIEW

**Comment by vhirtham** [comment]
Reminder: Fix second geometry tutorial

File: tutorials/geometry_02_geometry.ipynb [hunk 1]
```diff
@@ -470,9 +484,16 @@
     "trace_custom_interp = Trace(LinearHorizontalTraceSegment(Q_(20, \"cm\")))\n",
     "geometry_custom_interp = Geometry(v_profile_custom_interp, trace_custom_interp)\n",
     "\n",
-    "geometry_custom_interp.plot(profile_raster_width=Q_(1, \"cm\"),\n",
-    "                            trace_raster_width=Q_(2, \"cm\"))"
+    "#geometry_custom_interp.plot(profile_raster_width=Q_(1, \"cm\"),\n",
+    "#                            trace_raster_width=Q_(2, \"cm\"))"
```
Comment (vhirtham) [inline tutorials/geometry_02_geometry.ipynb:488]: Undo

File: tutorials/geometry_02_geometry.ipynb [hunk 2]
```diff
@@ -383,8 +395,8 @@
     "                            )\n",
     "\n",
     "geometry_vp = Geometry(profile=v_profile,trace=trace)\n",
-    "geometry_vp.plot(profile_raster_width=Q_(1, \"cm\"),\n",
-    "                 trace_raster_width=Q_(8, \"cm\"))"
+    "#geometry_vp.plot(profile_raster_width=Q_(1, \"cm\"),\n",
+    "#                 trace_raster_width=Q_(8, \"cm\"))"
```
Comment (vhirtham) [inline tutorials/geometry_02_geometry.ipynb:399]: undo

**Comment by CagtayFabry** [comment]
I have made some slight updates to the plotting parameters in the welding examples

I think the tutorials are ready @vhirtham ? 🚀

**Comment by vhirtham** [comment]
@CagtayFabry almost, need to fix something in the geometry tutorials that is broken with the new code

File: weldx/geometry.py [hunk 3]
```diff
@@ -2197,6 +2207,9 @@ def plot(
         axes=None,
         fmt=None,
         set_axes_equal=False,
```
Comment (vhirtham) [inline weldx/geometry.py:2209]: remove

**Comment by vhirtham** [comment]
Only tests and Readme updates missing...

**Comment by CagtayFabry** [comment]
I will look through this tomorrow @vhirtham  :)

**Review by CagtayFabry — APPROVED** [review]

**Pull request merged**